contracts/test/FlashLoanReceiver01.sol

```diff
@@ -0,0 +1,27 @@
+pragma solidity =0.6.6;
+
+import '../libraries/SafeMath.sol';
+import '../interfaces/IERC20.sol';
+import '../interfaces/IFlashLoanReceiver.sol';
+import '../interfaces/IFlashLoanV1Pool.sol';
+
+contract FlashLoanReceiver01 is IFlashLoanReceiver {
```

Comment:
Why you create `FlashLoanReceiver01` which is based on `IFlashLoanReceiver
(actually FlashLoanReceiver02)`?

Reply:
version 01 is based on a higher version 02 is strange, isn't it?

Reply:
Done.